.circleci/config.yml

```diff
@@ -57,6 +57,26 @@ commands:
   non-coverage-tests:
     description: Do tests without tracking code coverage
     steps:
+      - run:
+          # This can fail sporadically, which is why we do it before the
```

Comment:
Why does this fail sporadically?
Is it a timeout? If so, can the timeout limit be increased?